bin/program.py

```diff
@@ -504,4 +505,4 @@ class Visualizer(Program):
     # Stdin and stdout are not used.
     def run(self, cwd, args=[]):
         assert self.run_command is not None
-        return exec_command(self.run_command + args, timeout=config.args.timeout or 30, cwd=cwd)
+        return exec_command(self.run_command + args, timeout=config.args.timeout, cwd=cwd)
```

Comment:
The `or DEFAULT` is needed here. Passing None removes the timeout

Reply:
but `config.args.timeout` should never be `None`? it should get a default value (at least with the new commit where I fixed my attempt to add it to `set_default_args`).